btc-lsp/src/BtcLsp/Class/Env.hs

```diff
@@ -26,6 +26,7 @@ class
   getGsEnv :: m GSEnv
   getLspPubKeyVar :: m (MVar Lnd.NodePubKey)
   getLspPubKey :: m Lnd.NodePubKey
+  getLspLndEnv :: m Lnd.LndEnv
```

Comment:
This method is redundant, let's remote it. There are already existing `withLnd` or `withLndT` which you can use instead.